components/bitcoin-topics/hero/Hero.tsx

```diff
@@ -34,182 +43,459 @@ function TrafficLightsIcon(props: React.ComponentPropsWithoutRef<"svg">) {
+                                                Last visit date
+                                            </p>
+                                        </div>
+                                    </div>
+                                </div>
                                 <Button
-                                    href="https://github.com/bitcoin-dev-project/bitcoin-topics"
-                                    target="_blank"
-                                    variant="secondary"
+                                    onClick={handleContinueReading}
+                                    className="w-full mt-4 bg-orange-500 hover:bg-orange-600 text-white font-semibold py-2 px-4 rounded-full transition duration-300 ease-in-out flex items-center justify-center text-sm"
                                 >
-                                    View on GitHub
+                                    Continue Learning
+                                    <ArrowRightIcon className="ml-2 h-4 w-4" />
                                 </Button>
                             </div>
                         </div>
-                    </div>
-                    <div className="relative lg:static xl:pl-10">
-                        <div className="absolute inset-x-[-50vw] -bottom-48 -top-32 [mask-image:linear-gradient(transparent,white,white)] lg:-bottom-32 lg:-top-32 lg:left-[calc(50%+14rem)] lg:right-0  [mask-image:linear-gradient(transparent,white,transparent)] lg:[mask-image:linear-gradient(white,white,transparent)]">
-                            <HeroBackground className="absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2 lg:left-0 lg:translate-x-0 lg:translate-y-[-60%]" />
-                        </div>
-                        <div className="relative">
-                            <Image
-                                className="absolute -right-64 -top-64"
-                                src={blurOrangeImage}
-                                alt="Blur orange image"
-                                width={530}
-                                height={530}
-                                unoptimized
-                                priority
-                            />
-                            <Image
-                                className="absolute -bottom-40 -right-44"
-                                src={blurIndigoImage}
-                                alt="Blur indigo image"
-                                width={567}
-                                height={567}
-                                unoptimized
-                                priority
+
+                        {/* Improved Discord card */}
+                        <motion.div
+                            className="relative overflow-hidden cursor-pointer group"
+                            whileTap={{ scale: 0.98 }}
+                            onClick={() =>
+                                window.open(
```

Comment:
Would an <a> </a> or <Link></Link> tag work in this scenario rather than using the global window object ?
If it would please replace

Reply:
fixed here https://github.com/bitcoin-dev-project/bitcoin-dev-project/pull/102/commits/f9a0cdc61d5166831306b155dddc90a4b186f7b5